static/app/views/onboarding/targetedOnboarding/onboarding.tsx

```diff
@@ -63,6 +64,7 @@ function Onboarding(props: Props) {
     params: {step: stepId},
   } = props;
   const cornerVariantTimeoutRed = useRef<number | undefined>(undefined);
+  const [clientState, setClientState] = usePersistedOnboardingState();
```

Comment:
@Neo-Zhixing would it make sense to return `setClientField` function instead of `setClientState` since we only set one field at a time?

Reply:
If we want to iterate on this it'll probably make more sense to make usePersistedOnboardingState return an `updateClientState`. I wouldn't worry about that for now.